app/src/main/java/io/github/droidkaigi/confsched/activity/SearchActivity.java

```diff
@@ -201,6 +206,14 @@ public void afterTextChanged(Editable s) {
         }
     }
 
+    @Override public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
+        if (actionId == EditorInfo.IME_ACTION_SEARCH) {
+            InputMethodManager imm = (InputMethodManager)getSystemService(Context.INPUT_METHOD_SERVICE);
```

Comment:
I think we can also provide InputMethodManager by Dagger but what do you think @konifar?